.circleci/config.yml

```diff
@@ -7,14 +7,15 @@ orbs:
 executors:
   apple-ci-arm-medium:
     macos:
-      xcode: 14.3.1
+      xcode: 15.4.0
     resource_class: macos.m1.medium.gen1
 
 commands:
   install_android_sdk:
     description: "Install Android SDK in macOS machine"
     steps:
-      - run: brew install wget
+      - run: brew install wget openjdk@17
+      - run: sudo ln -sfn /opt/homebrew/opt/openjdk\@17/libexec/openjdk.jdk /Library/Java/JavaVirtualMachines/openjdk.jdk
```

Comment:
I didn't need this yesterday, but for some reason gradle complains that it can't find an appropriate java version when I ran this job today.